include/uv-win.h

```diff
@@ -45,8 +45,8 @@ typedef struct pollfd {
 
 #ifndef _malloca
 # if defined(_DEBUG)
-#  define _malloca(size) malloc(size)
-#  define _freea(ptr) free(ptr)
+#  define _malloca(size) uv__malloc(size)
+#  define _freea(ptr) uv__free(ptr)
```

Comment:
Oh wow, didn't realize that was a public header.

It looks like the only code inside libuv that's using _malloca is a single function in src/win/process.c.  So this could either be moved to process.c, or the code could just always do a heap allocation.  (I doubt the stack allocation is saving much as far as this optimization goes.)